cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/distributed/AbstractProgramTwillRunnable.java

```diff
@@ -168,9 +170,12 @@ public void initialize(TwillContext context) {
       metricsCollectionService = injector.getInstance(MetricsCollectionService.class);
       streamCoordinatorClient = injector.getInstance(StreamCoordinatorClient.class);
 
+      programOpts = createProgramOptions(cmdLine, context, context.getSpecification().getConfigs());
+
       // Initialize log appender
       logAppenderInitializer = injector.getInstance(LogAppenderInitializer.class);
       logAppenderInitializer.initialize();
+      SystemArguments.setLogLevel(resolveScope(programOpts), logAppenderInitializer);
```

Comment:
I am calling resolveScope() in createProgramOptions() method, so I think we do not need to call it here, I will remove it